src/main/java/de/tum/in/www1/artemis/web/rest/dto/pageablesearch/BasePageableSearchDTO.java

```diff
@@ -0,0 +1,67 @@
+package de.tum.in.www1.artemis.web.rest.dto.pageablesearch;
+
+import de.tum.in.www1.artemis.domain.enumeration.SortingOrder;
+import de.tum.in.www1.artemis.web.rest.dto.SearchResultPageDTO;
+
+// TODO (follow-up, as it changes 40+ files) rename this, move & rename PageableSearchDTO, move UserPageableSearchDTO.
```

Comment:
Open a separate PR. Since this can be auto refactored, we don't need the whole review process 👍